src/charts/brush.js

```diff
@@ -110,11 +113,11 @@ export default function Brush (_container) {
       return
     }
 
-    const extentClamped = getDataExtentUnderBrush()
+    let extent = getDataExtentUnderBrush()
```

Comment:
All of the changes from here on appear to be unnecessary?

Reply:
just changing `extentClamped` to `extent` to reflect that it will not always be clamped.